Src/Sarif.PatternMatcher/AnalyzeOptions.cs

```diff
@@ -15,6 +15,7 @@ public class AnalyzeOptions : AnalyzeOptionsBase
         [Option(
             'd',
             "search-definitions",
+            Separator = ';',
```

Comment:
>Separator [](start = 12, length = 9)

currently to make this argument work, you need it at the very end of the command-line and you provide a space separatow. Now you can embed this arg anywhere on the conmand-line, just use the ; to separate directories. this is the separator for windows %PATH% var, so a good one, obviously for this purpose.